docs/.vuepress/styles/palette.scss

```diff
@@ -0,0 +1,53 @@
+/*
+palette.scss is used for CSS variable injecting, the following are supported configurations and default values.
+
+
+Color Config:
+  For all colors, if they are the same in light mode and dark mode, you can set them directly; otherwise, please set a Sass variable of type Map to give the color values in light and dark modes respectively.
+
+  Available color variables:
+  - $theme-color: theme color
+  - $text-color: text color
+  - $bg-color: background color
+  - $bg-color-light: lighter background color
+  - $bg-color-active: background color using on active elements
+  - $bg-color-back: background color using on mask or background elements
+  - $bg-color-float: float background color using on floating elements
+  - $bg-color-blur: background color using on blur elements
+  - $bg-color-float-blur: background color using on floating blur elements$border-color: border color
+  - $box-shadow: shadow color using on elements
+  - $card-shadow: shadow color using on cards
+
+Layout Config
+  Available layout variables:
+  - Navbar:
+   -$navbar-height: navbar height
+   - $navbar-horizontal-padding: navbar horizontal padding
+   - $navbar-vertical-padding: navbar vertical padding
+   - $navbar-mobile-height: navbar height on mobile devices
+   - $navbar-mobile-horizontal-padding: navbar horizontal padding on mobile
+  - $navbar-mobile-vertical-padding: navbar vertical padding on mobile
+   - Sidebar:
+    - $sidebar-width: sidebar width
+    - $sidebar-mobile-width: sidebar width on mobile
+  Content:
+    - $content-width: width of main content
+    - $home-page-width: width of homepage content
+  Fonts:
+    - $font-family: font family used on normal text
+    - $font-family-fancy: font family used on fancy elements
+  Code:
+    - $font-family-code: font family used on code
+    - $line-numbers-width: width of line number in code blocks
+  Transition:
+    - $color-transition: transition used on colors
+    - $transform-transition: transition used on transform animation
+*/
+
+@font-face {
+  font-family: 'Jura';
+  src: url('https://fonts.googleapis.com/css2?family=Jura:wght@500&display=swap');
+}
```

Comment:
I think this might be the problem behind:
```
downloadable font: rejected by sanitizer (font-family: "Jura" style:normal weight:400 stretch:100 src index:0) source: https://fonts.googleapis.com/css2?family=Jura:wght@500&display=swap 
```
I think this file is only for variable injecting? Does this need to go in `index.scss`?

Reply:
Well, moving it to index didn't fix that either.

Reply:
Never mind I think I fixed it